src/models/AppState.js

```diff
@@ -175,6 +175,15 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
           return factors && factors.length > 1;
         }
       },
+      'userId': {
+        deps: ['lastAuthResponse'],
```

Comment:
I was wondering if we get a userId when we hit PASSWORD_WARN or PASSWORD_EXPIRED, and it looks like we do. So we're good to use userId (in case that came up).